web/client/components/misc/__tests__/Stepper-test.jsx

```diff
@@ -36,6 +37,38 @@ describe('Stepper component', () => {
         expect(footerStepBar).toExist();
         expect(footerStepBar.childElementCount).toBe(1);
     });
+
+    it('Stepper clickable on edit', () => {
+        const container = document.getElementById("container");
+        const enableClickOnStep = true;
+        const steps = [{
+            id: 'test',
+            label: 'test',
+            component: <div className="test-component"></div>
+        },
+        {
+            id: 'test2',
+            label: 'test2',
+            component: <div className="test-component"></div>
+        }
+        ];
+        let currentStep = steps[0];
+        const onSetStep = (stepId) => {
+            for (let i = 0; i < steps.length; i++) {
+                if (steps[i].id === stepId) {
+                    currentStep = steps[i];
+                }
+            }
+        };
```

Comment:
I think It would be better to use the done function and execute the expect directly in the callback instead of assign the currentStep varaible so you can remove the `expect(currentStep.id).toBe('test2');` outside of the callback.
With the done argument the test should wait until the done() dispatch unless there are error or a timeout. 
```js
it('Stepper clickable on edit', (done) => {
   // ...
   onSetStep  = (stepId) => {
      expect(stepId).toBe('test2');
      done();
   };
});
```